src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.darts.audio.service.impl;
+
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.darts.audio.entity.MediaRequest;
+import uk.gov.hmcts.darts.audio.repository.MediaRequestRepository;
+import uk.gov.hmcts.darts.audio.service.AudioTransformationService;
+
+import static uk.gov.hmcts.darts.audio.enums.AudioRequestStatus.PROCESSING;
+
+@Service
+@RequiredArgsConstructor
+public class AudioTransformationServiceImpl implements AudioTransformationService {
+
+    private final MediaRequestRepository mediaRequestRepository;
+
+    @Transactional
```

Comment:
Does this need to be Transactional?
My understanding is that as Keda is spinning up a pod per request, there shouldn't be any concurrent access.

Reply:
I thought it should be similar to `createCourtoom`. I'm not sure what else will be added to the `processAudioRequest`, it seemed obvious candidate for marking as `Transactional`. It appears that removing the annotation here still works currently with `AudioTransformationServiceTest`, but I did need it added in other places for the Tests to run `- no Session`

Reply:
createCourtroom could be accessed by multiple means in the future, e.g. getCases, addEvent, addAudio etc. And so, especially with event, we could get a flurry of similar ones at once, and so if courtroom doesn't exist, we only want it to create 1 new one, and not multiple identical ones.

Reply:
- I've kept the `@Transactional` on this top level service, since it's updating the database.
- Added `@EnableTransactionManagement` to the Application class.
- After changing the `getMediaRequestById` to use `mediaRequestRepository.findById`, I was able to re-enable the failing test I disabled yesterday in `MediaRequestServiceTest`. 
- Also removed the `@Transactional` added yesterday for the Tests to run `- no Session`.